src/main/java/com/amazonaws/kinesisvideo/demoapp/activity/WebRtcActivity.java

```diff
@@ -194,6 +201,9 @@ public void onSdpAnswer(final Event answerEvent) {
                 final SessionDescription sdpAnswer = new SessionDescription(SessionDescription.Type.ANSWER, sdp);
 
                 localPeer.setRemoteDescription(new KinesisVideoSdpObserver(), sdpAnswer);
+                peerConnectionFoundMap.put(answerEvent.getSenderClientId(), true);
```

Comment:
We will use localPeer here but nothing prevents the application to support multi-peer at some stage.